### ISSUE TITLE
Missing text in manual

### ISSUE THREAD

**Issue by bcamp1973** [coherentgraphics/cpdf-binaries#3]
From [http://www.coherentpdf.com/cpdfmanual/node4.html](http://www.coherentpdf.com/cpdfmanual/node4.html)

```
The typical pattern for usage is (???)

and the simplest concrete example, assuming the existence of a file in.pdf is: (???)
```

**Comment by johnwhitington** [comment]
It looks like the latest upload of the HTML documentation did not include the images. For now, please refer to the PDF manual:

http://www.coherentpdf.com/cpdfmanual.pdf

**Comment by bcamp1973** [comment]
Great, thanks @johnwhitington!

**Comment by johnwhitington** [comment]
Noted to fix in tasks for uploading next version (2.3) out soon, so closing this.
